lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@smaeda-ks I've tried this on my devstack, by following steps:

1. Add `"PREVIEW_LMS_BASE": "preview1.localhost:8000"` to the `"FEATURES"` dict in (cms/lms).env.json files
2. Restart the lms & cms servers.

I was able to see `preview1` in studio and lms urls. I hope you are doing this in devstack and properly configured you localhost to support `preview` feature as mentioned [here](http://edx.readthedocs.org/projects/edx-installing-configuring-and-running/en/latest/devstack/run_devstack.html#set-up-ability-to-preview-units-mac-linux-only)

Please let me know if it's still not working for you. Thanks.